extensions/stackdriver/edges/edge_reporter.cc

```diff
@@ -49,10 +49,16 @@ void instanceFromMetadata(const ::Wasm::Common::FlatNode& node_info,
   auto namespace_ = node_info.namespace_()
                         ? node_info.namespace_()->string_view()
                         : absl::string_view();
-  if (name.size() > 0 && namespace_.size() > 0) {
-    absl::StrAppend(instance->mutable_uid(), "kubernetes://", name, ".",
-                    namespace_);
+
+  if (Common::isRawGCEInstance(node_info)) {
+    instance->set_uid(Common::getGCEInstanceUID(node_info));
+  } else {
```

Comment:
You can do `else if () {..}` instead of `else { if () {..} }` here.

Reply:
yup. a holdover from more complex logic there. done.